tutorials/hyperspectral/Plot-Hyperspectral-Spectra.Rmd

```diff
@@ -168,6 +154,7 @@ Now we're ready to plot our spectral profile!
 
```

Comment:
Thoughts on moving away from qplot and putting everything in plot or ggplot?  I'd heard someone at ESA last summer say qplot is disappearing but unable to confirm though an internet search.

Reply:
Oh, I haven't heard that, and honestly I would be surprised. I like it here because it gets rid of several steps that don't add value to this particular lesson. I would be fine with converting to ggplot if you have an opinion on it, but I don't see this as a need. The next tutorial (currently called advanced-spectral-signatures) does the plotting with much nicer ggplot formatting because the advanced features of ggplot are required to visualize things nicely.

Reply:
Yes, change this s qplot may be depricated.

Reply:
Replace with ggplot.

Reply:
Made this change in commit a819a1d